src/main/java/gov/nih/nci/evs/api/service/ElasticSearchServiceImpl.java

```diff
@@ -97,6 +97,9 @@ public ConceptResultList search(SearchCriteria searchCriteria) throws Exception
                   .queryStringQuery("synonyms.normName:" + normTerm).analyzeWildcard(true),
                   ScoreMode.Max).boost(20f));
 
```

Comment:
I think here you want to do something more like:

      if (startsWithFlag) {
        boolQuery2 = boolQuery2
            .should(QueryBuilders.matchQuery("normName", normTerm.replaceFirst("\\*$", "")))
            .boost(20f);
      }

We're looking for an "exact" match to the string (without the *) (so we want "matchQuery" instead of "termQuery") and we also want to make sure this logic bears out for things with spaces.    So in addition to a test for "blood" do a test for "malignant neoplasm" because there are a bunch of concepts that start with "malignant neoplasm" but have other stuff afterwards, AND "malignant neoplasm" is its own concept.